stetl/filters/templatingfilter.py

```diff
@@ -94,6 +94,13 @@ class StringTemplatingFilter(TemplatingFilter):
     consumes=FORMAT.record or FORMAT.record_array, produces=FORMAT.string
     """
 
+    @Config(ptype=bool, default=False, required=False)
+    def safe_substitution(self):
+        """
+        Apply safe substitution?
```

Comment:
Possibly add more comment (I did not know e.g. about this standard option in Python Templates), like 
*if placeholders are missing from mapping and keywords, instead of raising an exception, the original placeholder will appear in the resulting string intact.*

Reply:
Fair point. Usually I don't add comments for things which can be easily looked up.